### DISCUSSION TITLE
Mixed Economy with CMI and PointsAPI

### DISCUSSION THREAD

**Discussion by Someone-NEW** [Zrips/CMI#869] [Feature Requests]
Zrips i found problem/bug idk. but i cant create /res create TEST , for example i have /money enough, but i have a separate system for Coins --> /points "with **PointsAPI**" form BukkitDEV from here (https://www.mine-home.net/products) and something "economy" is mixed and when i create residence take my Coins and not Survival "from **CMI Economy** "money. I use your Vault, CMI and economy injector. What is here possible to do? I just need to be separate and Residence wil take money from right Economy system. "Using per world too"
Please can you look around for this problem i have? maybe some new sugestion about this to take it to high level update.

I use per world system and Spigot 1.12.2
Im SomeonePIX user from Spigot

Thank you.

**Comment by CloudeLecaw** [comment]
this seems to be the wrong section. Resisdence Plugin has ist own area.. pls close this post and post the Problem there.

> I use your Vault, CMI and economy injector. 

Maybe you noticed a Little error?...
Two ways to use the economey..

1. Orgiginal VAULT + CMIInjector + CMI
OR (and not AND!!!)
2. Zrips VAULT on CMISite + CMI

just Option 1 or Option 2... No mixing of Option one and two! ...this is what you did..,

**Comment by Someone-NEW** [comment]
You are wrong, not understand what is question or problem. I just wait for response Zrips to answer about this problem and see what they can do about this. But no luck
And a suggestion for per world economy will be usefull too.

My Residence take money from other economy system wich use named Coins using PointsAPI and not from CMI Economy...........
